### PR TITLE
Header height, tweaks and new button in the mobile header

### PR DESCRIPTION
**Pull request by hedvigoscar**

### CODE REVIEW

**Comment by hedvigoscar** [comment]
Screenshot 2019-07-16 at 14 31 26

File: src/blocks/HeaderBlock/mobile.tsx [hunk 1]
```diff
@@ -52,28 +55,26 @@ export const CrossBurger = styled('div')(
...
-  position: 'absolute',
   width: '100%',
   top: '50%',
   left: 0,
   right: 0,
   height: 3,
   backgroundColor: isOpen && !isClosing ? 'transparent' : 'currentColor',
   transition: 'background-color 300ms',
-  transform: 'translateY(-1.5px)',
+  transform: 'translateY(-1px)',
```
Comment (palmenhq) [inline src/blocks/HeaderBlock/mobile.tsx]: when i come to think of it i think this should be half of `height` which is 3 and that's why it was 1.5. any particular reason for changing it?

File: app.json [hunk 2]
```diff
@@ -28,4 +34,4 @@
       "url": "heroku/nodejs"
     }
   ]
-}
+}
```
Comment (palmenhq) [inline app.json]: ):

File: src/blocks/HeaderBlock/mobile.tsx (hunk 1 above)
Comment (hedvigoscar) [reply src/blocks/HeaderBlock/mobile.tsx]: This is the half-pixel thing that made it look off on different screens. I guess we could try to cover it by using half-pixels on the other places as well? But this looks correct I think 🤷‍♀

File: app.json (hunk 2 above)
Comment (hedvigoscar) [reply app.json]: I have no idea why it did this 😞

**Pull request merged**